android/src/main/java/com/ryanheise/audioservice/AudioService.java

```diff
@@ -251,16 +251,14 @@ void setState(List<NotificationCompat.Action> actions, int actionBits, int[] com
 
     public int getPlaybackState() {
         switch (processingState) {
-        case none: return PlaybackStateCompat.STATE_NONE;
         case connecting: return PlaybackStateCompat.STATE_CONNECTING;
-        case ready: return playing ? PlaybackStateCompat.STATE_PLAYING : PlaybackStateCompat.STATE_PAUSED;
+        case ready: case completed: return playing ? PlaybackStateCompat.STATE_PLAYING : PlaybackStateCompat.STATE_PAUSED;
```

Comment:
The `completed` case, though reasonable, looks like a breaking change. Do you predict any negative impact from changing this?

An alternative is to fix it in the `one-isolate` branch.

Reply:
It is not a breaking change, neither a behaviour change also, because that was already the logic in line 263.
I just changed it to a cascading case because of a lint warning, since it could be simplified.
I didn't want to create a separate PR just for that change. Not worth it IMO.

Reply:
Ah, you're right. I didn't check the full context of the diff but it is indeed a logic-preserving change.

I'll be happy to merge it after giving it a day or two to see if any issues pop up.

Reply:
No problem. I'll release a version of my app using the forked repo and will be paying attention during the first days after the release for potential issues.

Reply:
Hi @letiagoalves , I assume things have been going smoothly?

Reply:
Any updates after testing?

Reply:
FWIW this is already in one-isolate and I haven't seen any negative effects

Reply:
Oh I misread the code it's just a refactor, shouldn't affect anything